components/Card.js

```diff
@@ -1,38 +1,185 @@
 import styled from "styled-components";
 import { useState } from "react";
-import { Button } from "./Button";
+import iconArrowRight from "../public/assets/voc-trainer_icon_arrow-right.svg";
+import iconEye from "../public/assets/voc-trainer_icon_eye.png";
+import iconCheckmark from "../public/assets/voc-trainer_icon_checkmark.png";
+import iconCross from "../public/assets/voc-trainer_icon_cross.png";
 
-export default function Card({ voc, handleHit, handleView }) {
+import Image from "next/image";
+
+export default function Card({ voc, handleHit, handleView, cardColor }) {
   const [showTranslation, setShowTranslation] = useState(false);
+  const [wasClicked, setWasClicked] = useState(false);
+
+  function handleSliderClick() {
+    setShowTranslation(!showTranslation);
+    handleView(voc.id);
+  }
+
+  function handleCorrectClick() {
+    setWasClicked(true);
+    handleHit(voc.id);
+  }
+
+  function handleWrongClick() {
+    setWasClicked(true);
+  }
 
   return (
-    <StyledSection>
-      <p style={{ whiteSpace: "pre-wrap" }}>
-        {voc.word} {voc.views} {voc.hits}
-        {showTranslation && `\n = \n ${voc.translation}`}
-      </p>
-      {!showTranslation && (
-        <Button
-          onClick={() => {
-            handleHit(voc.id);
-            handleView(voc.id);
-            setShowTranslation(
-              (previousShowTranslation) => !previousShowTranslation
-            );
-          }}
-        >
-          Show answer
-        </Button>
+    <StyledCard>
+      <Word>{voc.word}</Word>
+      <ViewsCount aria-label="Number of views:">
+        <ViewsIcon>
+          <Image src={iconEye} alt="Eye icon" width="18" />
+        </ViewsIcon>
+        {voc.views}
+      </ViewsCount>
+      <HitsCount aria-label="Number of hits:">
+        <HitsIcon>
+          <Image src={iconCheckmark} alt="Checkmark icon" width="16" />
+        </HitsIcon>
+        {voc.hits}
+      </HitsCount>
+      <HorizontalLine />
+      <Translation
+        className={showTranslation ? "" : "hidden"}
```

Comment:
I was wondering about ``className=``. Isn't it better to use props instead since we're using styled-components? We could use the prop ``showTranslation={showTranslation}`` and then have to refactor the ``Translation``-component of cause. (Same we have to do with the slider-component)

Reply:
Of course we should use props instead! Since brackets are free of charge, there is nothing opposing this. 😉